<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSVFileControlWidget</class>
 <widget class="QWidget" name="CSVFileControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>24</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>24</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CSV File Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/images/calibrate-black.png</normaloff>:/images/calibrate-black.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0">
   <property name="spacing">
    <number>7</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="pathLabel">
     <property name="text">
      <string>File path: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="filePathLineEdit"/>
   </item>
   <item>
    <widget class="QToolButton" name="browseToolButton">
     <property name="text">
      <string>Select</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/images/open-black.png</normaloff>:/images/open-black.png</iconset>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="processToolButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{color:red;}</string>
     </property>
     <property name="text">
      <string>Process</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
